src/scales/scale.time.js

```diff
@@ -76,6 +76,9 @@ module.exports = function(Chart) {
 			Chart.Scale.prototype.initialize.call(this);
 		},
 		getLabelMoment: function(datasetIndex, index) {
+			if (datasetIndex === null || index === null)
```

Comment:
Try to stick to the surrounded coding style: 3 lines under there is a single statement if using brackets.... so please, always use brackets :)